Tests/ApolloCodegenTests/CodeGeneration/Templates/FragmentTemplateTests.swift

```diff
@@ -88,6 +88,42 @@ class FragmentTemplateTests: XCTestCase {
     expect(String(actual.reversed())).to(equalLineByLine("}", ignoringExtraLines: true))
   }
 
+  func test__render__givenLowercaseFragment_generatesTitleCaseTypeName() throws {
+    // given
+    document = """
+    fragment testFragment on Query {
+      allAnimals {
+        species
+      }
+    }
+    """
+
+    let expected =
+    """
+    public struct TestFragment: TestSchema.SelectionSet, Fragment {
+      public static var fragmentDefinition: StaticString { ""\"
+        fragment testFragment on Query {
+          __typename
```

Comment:
This test probably doesn't need anything beyond the first three lines of the expected output; matching TitleCase struct name and ensuring that the definition is preserved with the correct casing. The rest isn't important to the test passing and would get swallowed by the `ignoringExtraLines` param on `equalLineByLine`.

Doesn't need to change for this test, just a handy tip for it you ever need to write tests using that matcher again.